renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -1435,3 +1435,12 @@ extern "C" RENDERDOC_API void RENDERDOC_CC RENDERDOC_EnumerateAndroidDevices(rdc
 
 DOCUMENT("Internal function for starting an android remote server.");
 extern "C" RENDERDOC_API void RENDERDOC_CC RENDERDOC_StartAndroidRemoteServer(const char *device);
+
+DOCUMENT("Internal function for checking for presence of RenderDoc Vulkan layer on remote device.");
+extern "C" RENDERDOC_API bool RENDERDOC_CC RENDERDOC_CheckAndroidLayer(const char *host,
+                                                                       const char *exe);
```

Comment:
Yeah, I'll rename and incorporate the desktop flag flow.  Thanks for pointing that out.